docs/guides/development.mdx

```diff
@@ -9,23 +9,40 @@ order: 10
-Prettier, on the other hand, takes some of the most menial tasks and performs them automatically. Spacing, import order etc. are pretty easy to arrange for a machine, so that's what it does. Again, there are a lot of prettier plugins. It might save you a lot of time to enable pretter to run on file-save so you never save a badly formatted file. 
+
+Minds can collide over how to best structure code. Architecturally, that may still be the case, but it doesn't have to be that way on a small scale. Eslint and Tslint declare some code styling rules we commited to and want to follow (for .js and .ts respectively) and those rules are defined in the .eslintrc and tslint.json. There are many extensions for you favorite code editor that will read these files automatically and show you the warnings/errors while coding.
+Prettier, on the other hand, takes some of the most menial tasks and performs them automatically. Spacing, import order etc. are pretty easy to arrange for a machine, so that's what it does. Again, there are a lot of prettier plugins. It might save you a lot of time to enable pretter to run on file-save so you never save a badly formatted file.
 
 ## Before you pull request:
-Please make sure that you have no linting and typing errors left in your code and explain the hard parts with comments. 
+
+Please make sure that you have no linting and typing errors left in your code and explain the hard parts with comments.
+
+## edtrio-client
+
+### Styled-Components
+
+We really want to avoid normal css files with a global namespace, since the editor will be used inside the schul-cloud in the future.
+For styling we therefore recommend using [styled-components](https://www.styled-components.com/). This means no more css files you have to import, every styled component becomes easily exportable and automatically ships with its own styles. Moreover, it more easily allows for adding variables to your styling. Check it out, you won't regret it.
+
+## edtrio-server
+
+### getting started
+
+This is a GraphQl Server built with [graphql-yoga](https://github.com/prisma/graphql-yoga). You can imagine yoga being the create-react-app for graphql servers. If you have no clue what graphql is, head on over to [this awesome tutorial](https://www.howtographql.com/).
+In order to get the server up and running, install [Docker](https://store.docker.com/editions/community/) and [prisma](https://www.prisma.io/) (preferably installed globally via npm or yarn).
+You can then run the `yarn run startFreshDB` script in the edtrio-server package. This will hop over to the src/database folder, start the docker image for the database, generate the [prisma client](https://www.prisma.io/docs/prisma-client/) code, reset the content of the database and seed it. If you ever need to give more detailed commands to the prisma instance, manually head over to src/datbase with a terminal and use the prisma cli.
```

Comment:
```suggestion
You can then run the `yarn run startFreshDB` script in the edtrio-server package. This will hop over to the `src/database` folder, start the docker image for the database, generate the [prisma client](https://www.prisma.io/docs/prisma-client/) code, reset the content of the database and seed it. If you ever need to give more detailed commands to the prisma instance, manually head over to `src/database` with a terminal and use the prisma cli.
```